### PR TITLE
Add iS= and iSq.

### PR DESCRIPTION
**Pull request by Maijin**

### CODE REVIEW

File: libr/core/cbin.c [hunk 1]
```diff
@@ -2197,6 +2281,11 @@ static int bin_sections(RCore *r, int mode, ut64 laddr, int va, ut64 at, const c
 	if (chksum && *chksum == '.') {
 		printHere = true;
 	}
+	if (IS_MODE_EQUAL(mode)) {
+		int cols = r_cons_get_size (NULL);
+		list_section_visual(r->io, sections, r->offset, -1, r->print->flags & R_PRINT_FLAGS_COLOR, cols);
```
Comment (radare) [inline libr/core/cbin.c]: missing space

File: libr/core/cbin.c [hunk 2]
```diff
@@ -2197,6 +2281,11 @@ static int bin_sections(RCore *r, int mode, ut64 laddr, int va, ut64 at, const c
 	if (chksum && *chksum == '.') {
 		printHere = true;
 	}
+	if (IS_MODE_EQUAL(mode)) {
```
Comment (radare) [inline libr/core/cbin.c]: missing space

File: libr/core/cbin.c (hunk 1 above)
Comment (ret2libc) [reply libr/core/cbin.c]: just try `./sys/clang-format-diff.py` please.

**Comment by radare** [comment]
broken tests are unrelated please rebase

File: libr/core/cbin.c (hunk 1 above)
Comment (radare) [reply libr/core/cbin.c]: r2pm -i r2i

**Pull request merged**